apps/lib/s_socket.c

```diff
@@ -26,6 +26,14 @@
 typedef unsigned int u_int;
 #endif
 
+#ifdef _WIN32
+/*
+ * With MSVC, certain POSIX functions have been renamed to have an underscore
```

Comment:
I assume this is particular versions that need this?

Reply:
Nope.